.github/workflows/docker-test.yml

```
@@ -4,6 +4,9 @@ on:
   pull_request:
     branches:
       - main
+      - develop
+    tags:
```

actionlint (1.7.8)

"tags" filter is not available for pull_request event. it is only for push event

(events)